packages/sveltekit/README.md

```diff
@@ -222,37 +222,26 @@ For [row level security](https://supabase.com/docs/learn/auth-deep-dive/auth-row
 ```ts
 // src/routes/profile.ts
 import { supabaseServerClient, withApiAuth } from '@supabase/auth-helpers-sveltekit';
-import type { RequestHandler } from './__types/profile';
+import type { RequestHandler } from './__types/protected-route';
```

Comment:
Why did you change the type origin to `protected-route`? Aren't we on the file `profile.ts`?